docs/userguide.md

```diff
@@ -93,6 +93,12 @@ automated, then invoke the deployment script by running:
 ./scripts/deploy-federation-latest.sh cluster2
 ```
 
+The above script joins the host cluster to the federation control plane it deploys, by default.
+The argument used is the context name of the additional clusters that needs to be joined to this
```

Comment:
@gyliu513 apologies, While reading it before approving; I found the english mistake.
This should be updated as:
`The argument(s) used is/are the list of context names of the...`.
Please update and I will merge, unless @font has any further comments.

Reply:
Done